### PR TITLE
Later jquery

### PR DESCRIPTION
**Pull request by mmaiza**
This pull adds support for later versions of jquery as well as fixes ie issue and adds flexibility in setting tags. Please verify and merge we need it to complete ag sites move to bower.

### CODE REVIEW

File: src/jquery.autotagging.coffee [hunk 1]
```diff
@@ -1,20 +1,4 @@
...
-      if fromIndex < 0
-        fromIndex += length
-        fromIndex = 0  if fromIndex < 0
-      while fromIndex < length
-        return fromIndex  if this[fromIndex] is searchElement
-        fromIndex++
-      -1
-  # end polyfill
-
+define ['jquery', './lib/browserdetect', 'jquery-cookie-rjs',], ($, browserdetect) ->
```
Comment (c0) [inline src/jquery.autotagging.coffee]: For bower, this should keep `define ['jquery', 'browserdetect', 'jquery.cookie',], ($, browserdetect) ->`. ./lib/browserdetect and jquery-cookie-rjs are only used when it was jam.

File: src/jquery.autotagging.coffee [hunk 2]
```diff
@@ -142,6 +126,9 @@ define ['jquery', 'browserdetect', 'jquery.cookie',], ($, browserdetect) ->
       obj.registration            = if $.cookie('sgn') == '1' then 1 else 0
       obj.person_id               = $.cookie('zid') if $.cookie('sgn')?
 
+      @metaData.cg = obj.cg if obj.cg?
```
Comment (c0) [inline src/jquery.autotagging.coffee:129]: Can this be written as `@metaData.cg = if obj.cg? then obj.cg else ''`?

File: src/jquery.autotagging.coffee (hunk 1 above)
Comment (colinrymer) [inline src/jquery.autotagging.coffee]: The jam version now uses `define ['jquery', 'browserdetect', 'jquery.cookie',], ($, browserdetect) ->` as well. `jquery.cookie` and `browserdetect` have been uploaded to our jam server.

File: src/jquery.autotagging.coffee [hunk 3]
```diff
@@ -202,7 +189,8 @@ define ['jquery', 'browserdetect', 'jquery.cookie',], ($, browserdetect) ->
       klasses.split(' ')[0]
 
     getDataFromMetaTags: (obj) ->
-      retObj = { cg: '' }
+      #retObj = { cg: '' }
```
Comment (c0) [inline src/jquery.autotagging.coffee]: This line can be pruned.

File: src/jquery.autotagging.coffee (hunk 2 above)
Comment (mmaiza) [inline src/jquery.autotagging.coffee:129]: No it changes the logic metadata can be defined even when obj.cg is not.

**Pull request merged**